### PR TITLE
final README

### PR DESCRIPTION
**Pull request by pragatiahuja**
### Checklist

- [x] I've looked at the [contribution guidelines](https://github.com/raywenderlich/swift-algorithm-club/blob/master/.github/CONTRIBUTING.md).
- [x] This pull request is complete and ready for review.

### Description

Linked List is very commonly used with recursive approach as using recursion makes the process simpler and reduces the complexity of the code. So this is a code snippet of reversing a singly linked list using recursion. Hope it is useful!

### CODE REVIEW

**Comment by hollance** [comment]
Swift code doesn’t need semicolons at the end of statements.

File: Linked List/README.markdown [hunk 1]
```diff
@@ -482,6 +483,18 @@ How about reversing a list, so that the head becomes the tail and vice versa? Th
     }
   }
 ```
+Recursive Approach:
+```swift
+  public func reverse(node: head) {
```
Comment (richard-ash) [inline Linked List/README.markdown:488]:
```suggestion
  public func reverse(node: Head) {
```

File: Linked List/README.markdown [hunk 2]
```diff
@@ -482,6 +483,18 @@ How about reversing a list, so that the head becomes the tail and vice versa? Th
...
+    if (!head || !(head.next)) {
```
Comment (richard-ash) [inline Linked List/README.markdown:489]:
```suggestion
    if !head || !(head.next) {
```

Comment (richard-ash) [inline Linked List/README.markdown:489]:
```suggestion
    if !head || !head.next {
```

**Comment by richard-ash** [comment]
Thanks for the PR @pragatiahuja !

**Pull request merged**